slate/source/index.html.md

```diff
@@ -655,23 +672,23 @@ The following OIDC claims will be supported:
     -   `string`
     -   Available in ID Token Only.
     -   Authentication Context Class Reference.
-    -   If passed as an essential claim for the ID Token, the Authorization
-        Server must return an `acr` Claim Value that matches one of the requested
+    -   DR must include the `acr` claim as per
+        [FAPI Part 2 §5.2.3.3]](https://openid.net/specs/openid-financial-api-part-2.html#public-client)
```

Comment:
acr is not mandatory under FAPI Read profile https://openid.net/specs/openid-financial-api-part-1.html